### DISCUSSION TITLE
Is it possible to view the 'Languages' used in a repo in the app?

### DISCUSSION THREAD

**Discussion by new-creation7** [community/community#120189] [Mobile] [answered]
Is the visual representation of 'Languages' (Programming languages) used in a repo available on the app? . On the website it's available:
![languagesOnWeb.jpg](https://github.com/community/community/assets/161584341/a54278d9-87a2-40fc-9bf3-dd74a48ed982)

Should I close this discussion as it's related to https://github.com/orgs/community/discussions/34651 ?

**Comment by ghost** [comment]
GitHub's mobile app didn't offer a visual representation of the programming languages used in a repository. However, GitHub's website does provide this feature. You can view the breakdown of languages used in a repository by navigating to the repository's main page and selecting the "Languages" tab. This tab displays a pie chart showing the distribution of programming languages used in that particular repository. If there have been any updates or changes since then, you may want to check the latest documentation or release notes for GitHub's mobile app to see if this feature has been added.

**Comment by new-creation7** [comment]
> \"Languages\" tab

Apologies, Is that "Languages" tab you mentioned available in the app (Android) ?

Thank you.

**Comment by eddysantiagoo** [comment, marked as answer]
Hi @wencgg this feature is not available for the Android app. However, if you use a browser on your phone, you will be able to see it

If the answer helps you please mark this answer as the correct answer
